.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
would 22.04 be an option?

Reply:
Sorry, just realized you made this comment on `lint`, and there's no reason not to use `ubuntu-latest` here. As for the one below (with the matrix), we can probably just add `ubuntu-22.04` to the matrix. GUI tests can probably use a matrix as well, maybe with a couple of versions of Python as well.

I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1785095) to track this.